packages/components/src/theme/ComposerTheme/Components/Tag.ts

```diff
@@ -6,6 +6,7 @@ const baseStyle = {
     color: 'text-secondary',
     border: '0.0625rem solid var(--chakra-colors-ui-element-outline-default)',
     borderRadius: '0.375rem',
+
```

Comment:
😿 @Claratj can we remove this empty line?

Reply:
done!